pkg/cache/redis_test.go

```diff
@@ -66,7 +66,7 @@ func TestRedisCache_PutArtifact(t *testing.T) {
 				addr = "dummy:16379"
 			}
 
-			c, err := cache.NewRedisCache(fmt.Sprintf("redis://%s", addr), "", "", "", false, 0)
+			c, err := cache.NewRedisCache("redis://"+addr, "", "", "", false, 0)
```

Comment:
I feel like it's too much. `fmt.Sprintf` is easier to read than string concatenation.